### DISCUSSION TITLE
The places in the front end that need backend data

### DISCUSSION THREAD

**Discussion by e19166** [cepdnaclk/e19-CO227-A-Simple-Web-or-Mobile-Application-For-A-Gym#62] [Show and tell]
@hirushig 

- when the login button is pressed the user should logged in to their account
- In 'localhost:3000/trainerprofile' the logged in trainer name should be visible like "HI ......."
- for the table.jsx page all the appointments that are having for the logged in user and in that table (table is visible at 'localhost:3000/appointments') at the actions column there will be 2 buttons(Accept and decline). When the Accept is pressed the data should remain unchanged in the database and the trainee should get a email confirmation. When the decline is pressed the appointment data will be deleted from the database and the trainee get an email notification.
- In 'localhost:3000/trainersettings' the field with data that had collected at the registration should be filled already and then other details can be updated by the user. Then the updated data should go to the database when the user clicks the save button. And also when the trainer add a time slot, that should be stored in the database

**Comment by hirushig** [comment]
Register User - api/v1/registration/user
Timeslot
1. Add timeslot - api/v1/timeslot/add/{trainerId}
2. Get All Time Slots For Trainer - api/v1/timeslot//trainer/{trainerID}
3. Update Time Slot - api/v1/timeslot/update/{timeSlotId}
4. Delete Time Slot - api/v1/timeslot//delete/{timeSlotId}

@Wishula
